cli/src/config-schema.json

```
@@ -168,6 +168,16 @@
                     "type": "string",
                     "enum": ["none", "watchman"],
                     "description": "Whether to use an external filesystem monitor, useful for large repos"
+                },
+                "watchman": {
+                    "type": "object",
+                    "properties": {
+                      "set_up_trigger": {
```

```suggestion
                      "register_trigger": {
```

suggestion (non-blocking): I think "register" as you used elsewhere might be a slightly better term than "set up", in terms of formality and that it's easy to mix up the terms "set up" and "setup".